src/renderer/protocol-handler/app-handlers.ts

```diff
@@ -54,5 +55,15 @@ export function bindProtocolAddRouteHandlers() {
     })
     .addInternalHandler("/extensions", () => {
       navigate(extensionsURL());
+    })
+    .addInternalHandler(`/extensions${LensProtocolRouter.ExtensionUrlSchema}`, ({ pathname, search: { version } }) => {
```

Comment:
Should route include "install", like `/extensions/install${LensProtocolRouter.ExtensionUrlSchema}`?

Reply:
I agree, might want to reserve /extensions/{extension-name} for some other behaviour